GliaWidgetsTests/SecureConversations/Welcome/SecureConversations.WelcomeViewController.Mock.swift

```diff
@@ -0,0 +1,37 @@
+#if DEBUG
```

Comment:
Do we need it in test target?

Reply:
@igorkravchenko I see that sometimes mocks are in the tests, but sometimes they are in the widgets themselves, and thus they have this `#if DEBUG` macro. Is there more of a recommended way to do this? Asking you because you are the environmental expert :)

Reply:
If I'm not mistaken, we discussed that would be good to have (or even move existing) mocks in test target. 
The only reason to keep them in WidgetsSDK target that I see is if we use some of them in both unit tests and snapshot tests targets to avoid duplications. But for this case we also can introduce additional target

Reply:
Thank you. I removed the macros and kept the mocks in the test target.